content/python/concepts/time-module/terms/time-strftime/time-strftime.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'time.strftime()'
+Description: 'Returns a time as a string based on one or more formatting codes.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Constants'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `strftime()` function returns a time string based on one or more formatting codes and a time represented as a tuple or `stuct_time` format.
+
+## Syntax
+
+```pseudo
+strftime(format_codes, time_value)
+```
+
+A combination of `format_codes` are used to format the returned string. The `time_value` is the time that will be formatted and can be represented as a [tuple](https://www.codecademy.com/resources/docs/python/tuples) or `struct_time` value. If no `time_value` is provided the function will return the current local time as formatted by the given expression.
+
+A few common codes are:
+
+- %a - The abbreviated day of the week (e.g. Mon)
+- %d - The day of month as two decimals (08)
+- %Y - The calendar year represented as four decimals (2020)
```

Comment:
The list of codes should be more complete. It should also definitely list the codes used in the examples.